### ISSUE TITLE
Add guests to appointments

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/calendar#3589]
**Is your feature request related to a problem? Please describe.**

As a user I want to add guests to the appointment slots that I book. The organizer will have to enable this option, though.

**Describe the solution you'd like**

An additional input field on the booking page. Guests will be added to the participants list, possibly requiring an RSVP and an invitation email.

**Describe alternatives you've considered**

N/a

**Additional context**

Follow-up to https://github.com/nextcloud/calendar/issues/1023

**Comment by kousu** [comment]
I want to use NextCloud to handle booking courses at a community center. I have different types of classes (pottery, dance, drawing, cooking, etc) and each individual class has a different attendance cap. This sounds like the feature I want but the description is a bit sparse so I want to check: would this allow multiple guests to sign up for classes in parallel? How do I find out who they are -- do I just get their emails listed on the attendance? How would I tell NextCloud to enforce my attendance cap? Can I have an option to manually confirm individual guests (e.g. so I can only let them in once they've paid their materials fees)?

By the way there is a competing app where this use-case [is also an open feature request](https://github.com/SergeyMosin/Appointments/issues/54).

**Comment by actuallyrisk** [comment]
i hope this gets implemented. Or at least define Groups to get automaticcly added into the Talk that gets created, would be nice.

**Comment by SebastianKrupinski** [comment]
Closing this in favour of https://github.com/nextcloud/calendar/issues/7596

**Comment by ChristophWurst** [comment]
But that is not the same. Guests mean that the booking party is explicitly adding someone to the booked slot. It's not implicitly a fully public appointment.

**Comment by kousu** [comment]
You can currently try to invite guests to a nextcloud calendar event, if you type in an email; they don't need appear to to need a nextcloud account:

<img width="480" height="572" alt="Image" src="https://github.com/user-attachments/assets/546b2358-fde5-422e-aa7e-3e746b6a21eb" />

the invitation ends up in their inbox

<img width="533" height="444" alt="Image" src="https://github.com/user-attachments/assets/9edd859c-77ca-4736-bad9-f3111d3c63f9" />

The accept/decline links are straightforward:

* `https://example.org/apps/dav/invitation/accept/${token}`
* `https://example.org/apps/dav/invitation/decline/${token}`

but clicking one fails

<img width="335" height="115" alt="Image" src="https://github.com/user-attachments/assets/710cac16-6dc3-4b74-8a6c-961b149adaa6" />

and I don't see anything in `nextcloud.log` about it.

Is this because people _need_ a nextcloud account? If so, removing that restriction is what this feature request is about.


The other proposal says invitees can forward a link for people to sign up. Presumably in that case, people won't need accounts, so fixing that would have to also fix this. But I agree with @ChristophWurst that invite-only (guest) attendees is a different feature than invite-**link** guest attendees. We need a middle ground between "members" and "public" events.

**Comment by ChristophWurst** [comment]
This FR is about the **appointments** feature, not regular events with attendees.
